README.md

```diff
@@ -90,6 +90,22 @@ sudo pip3 install rdiff-backup
 > **NOTE:** If your platform is not i386 or amd64, e.g. ARM/MIPS/etc,
   you may need the build dependencies `build-essentials`, `librsync-dev`.
 
+### CentOS and RHEL 6 (from Pypi)
+
+```
+yum install centos-release-scl
+yum install rh-python36 gcc libacl-devel
+ln -s /opt/rh/rh-python36/root/usr/bin/python /usr/bin/python3
+python3 --version
+
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
```

Comment:
According to https://www.softwarecollections.org/en/scls/rhscl/rh-python36/ I am the opinion that pip is installed once you've installed the collection, and that get-pip shouldn't be required. Do you confirm the need?

Reply:
Things seem a bit broken right now, but http://mirror.centos.org/centos/6/sclo/x86_64/rh/Packages/r/ conains pip for Python 3.5 and 3.6.